content/applications/finance/documents.rst

```diff
@@ -91,8 +106,88 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
      rights to the document. It is possible to replace the owner of a document. I.e.: an employee
      must be owner of a document to be able to see it in "My Profile".
 
-Finally, different :guilabel:`Actions` are available at the bottom of the right panel, depending on
-the workspace where your document is stored.
+Finally, different :ref:`Actions <documents/workflow-actions>` are available at the bottom of the
+right panel, depending on the workspace where your document is stored.
+
+Split documents
+---------------
+
+Select the document you want to split, and click the :guilabel:`Split` icon. A new page opens and
+displays all the pages of the document.
+
+The scissors between pages indicate where the split will occur. Click on the scissors if you
+don't want to split your document between two specific pages.
+
+.. image:: documents/split-pdf.png
+   :align: center
+   :alt: split your documents
+
+.. note::
+
+   To merge documents, select them from your dashboard and click the :guilabel:`Split` icon. Then,
+   click on the scissors between pages to remove the cut, and click :guilabel:`Split`. Your
+   documents are now merged.
+
+Additional features
+-------------------
+
+Select a specific workspace and click on the arrow next to the :guilabel:`Upload` button to access
+additional features:
+
+Request
+~~~~~~~
+
+To request a missing file, click :guilabel:`Request`. Add the :guilabel:`Document name`, select the
+person you need it from in the :guilabel:`Request to` field, fill in the :guilabel:`Due date`, add
+the :guilabel:`Workspace` the document should belong to, and eventually add :guilabel:`Tags` and a
+:guilabel:`Note`. A file reflecting the missing document is created in the workspace.
```

Comment:
the :guilabel:`Workspace` the document should belong to, and add :guilabel:`Tags` and a
:guilabel:`Note`, if needed.